plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -60,7 +66,23 @@ FileSystemAdaptor.prototype.getTiddlerFileInfo = function(tiddler,callback) {
 		});
 		$tw.boot.files[title] = fileInfo;
 	}
-	callback(null,fileInfo);
+	else if(fileInfo && fileSystemPaths) {
+		// Otherwise (check for `FileSystemPaths`), we'll need to (re)generate it
+		options.fileInfo = {...fileInfo};
```

Comment:
TW needs to be ES5 only so no Rest functionality.

Reply:
Thanks, changed this to explicitly map the properties.